src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindCommand.java

```diff
@@ -20,7 +20,7 @@
 @JsonTypeName(CommandName.Names.FIND)
 @CheckFindOption
 public record FindCommand(
-    @Valid @JsonProperty("filter") FilterClause filterClause,
+    @Valid @JsonProperty("filter") FilterSpec filterSpec,
```

Comment:
From discussions, try the @Schema.implementation property so that swagger generation works ?

Reply:
I added `@Schema` on `FilterSpec` itself, with `implementation = FilterClause.class` so that should work the same as before:

```
@Schema(
    type = SchemaType.OBJECT,
    implementation = FilterClause.class,
    example =
        """
                     {"name": "Aaron", "country": "US"}
                      """)
```
 
although I think the only useful part is probably `example` given that `FilterClause` structure does not (and cannot) match structurally JSON (because it's mostly wrapper around `LogicalExpression` etc).

But at least examples should be included in generated Swagger.

Reply:
cool, thinking ahead to this being a pattern it maybe useful - though most of the time we may be using this when it is as complex as the find